src/storage/src/source/generator/key_value.rs

```diff
@@ -51,140 +52,143 @@ pub fn render<G: Scope<Timestamp = MzOffset>>(
     let (_progress_output, progress_stream) = builder.new_output::<CapacityContainerBuilder<_>>();
     let (mut stats_output, stats_stream) = builder.new_output::<CapacityContainerBuilder<_>>();
 
-    let button = builder.build(move |caps| async move {
-        let [mut cap, mut progress_cap, stats_cap]: [_; 3] = caps.try_into().unwrap();
-
-        let resume_upper = Antichain::from_iter(
-            config.source_resume_uppers[&config.id]
-                .iter()
-                .map(MzOffset::decode_row),
-        );
-
-        let Some(resume_offset) = resume_upper.into_option() else {
-            return;
-        };
-
-        cap.downgrade(&resume_offset);
-        progress_cap.downgrade(&resume_offset);
-        start_signal.await;
+    let busy_signal = Arc::clone(&config.busy_signal);
```

Comment:
Same, this part also seems uncovered

Reply:
Yeah, this will be uncovered until we re-enable this test `load-generator-key-value.td`